.travis.yml

```diff
@@ -1,5 +1,5 @@
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
AFAIK, `Travis CI` doesn't yet handle Ubuntu Xenial. Instead, `industrial_ci` uses docker for testing ROS Kinetic on Xenial.